IR:CKII Compatibility

DrDudelsack opened the pull request.

IohannesIohannium commented:
What cultures and religions are added by I:R-CK2? I may handle them with the data files I added in #669 later on

IohannesIohannium commented on EU4toV2/Data_Files/configurables/culture_definitions.txt:
```diff
@@ -490,6 +490,52 @@ pre_indo_european = {
 			Punicid Orisid Amidian Nisunidian Tiresidian Budidian Gargoridian Culcidian Indicidian Iscerbelidian Karidian Korbidian Lagunidian Mandonidian Magavarician Uxentican Umarilid Turenidian Urceticidian
 		}
 	}
+	etrurian = {
```
The Etrurian name pool should use Etrurian rather than Latin names, same goes for Rhaetian

DrDudelsack replied:
I think grabbed the Etruscan names from EU IV, but that sounds good, when I get the chance I'll go ahead and give them Etruscan names from Imperator

DrDudelsack commented:
> What cultures and religions are added by I:R-CK2? I may handle them with the data files I added in #669 later on

I've added Sabellian, Old Ligurian, Venetic, Siculian, Etruscan, Rhaetian, Lepontic, Gallic, Noric (celtic_pannonian), Belgae, Punic, Phoenician, Dacian, Celtiberian, Lusitanian, Lycian, Phrygian, Isaurian, Paphlagonian, and Cilician, along with all culture mappings for them

IohannesIohannium commented:
> > What cultures and religions are added by I:R-CK2? I may handle them with the data files I added in #669 later on
> 
> I've added Sabellian, Old Ligurian, Venetic, Siculian, Etruscan, Rhaetian, Lepontic, Gallic, Noric (celtic_pannonian), Belgae, Punic, Phoenician, Dacian, Celtiberian, Lusitanian, Lycian, Phrygian, Isaurian, Paphlagonian, and Cilician, along with all culture mappings for them

Ok, Etruscan, Rhaetian, Gallic, Belgae, Punic, Phoenician, Dacian, Celtiberian, Lycian, Phrygian, Isaurian and Paphlagonian are all correctly set through #669; for the remaining ones, work is still to be done, but I added provisional mappings.

DrDudelsack commented:
> What cultures and religions are added by I:R-CK2? I may handle them with the data files I added in #669 later on

I haven't added religions or changed any religion mapping, I'm still working on creating them for the I:R-CK2 converter

> > > What cultures and religions are added by I:R-CK2? I may handle them with the data files I added in #669 later on
> > 
> > 
> > I've added Sabellian, Old Ligurian, Venetic, Siculian, Etruscan, Rhaetian, Lepontic, Gallic, Noric (celtic_pannonian), Belgae, Punic, Phoenician, Dacian, Celtiberian, Lusitanian, Lycian, Phrygian, Isaurian, Paphlagonian, and Cilician, along with all culture mappings for them
> 
> Ok, Etruscan, Rhaetian, Gallic, Belgae, Punic, Phoenician, Dacian, Celtiberian, Lycian, Phrygian, Isaurian and Paphlagonian are all correctly set through #669; for the remaining ones, work is still to be done, but I added provisional mappings.

That sounds good, in that case we could just add Sabellian, Old Ligurian, Venetic, Siculian, Lepontic, Noric (celtic_pannonian), Lusitanian, and Cilician from my branch to #669

IohannesIohannium commented:
For the religions, I suggest you take a peek at I:R-CK3 for inspiration. If you use the same strings, they'll also be already supported in EU4-Vic2.

Zemurin commented:
superseded by #672
